modules/develop/pages/data-transforms/build.adoc

```diff
@@ -358,15 +427,15 @@ use redpanda_transform_sdk::*;
 use redpanda_transform_sdk_sr::{SchemaRegistryClient};
 
 fn main() {
-    // Initialize the Schema Registry client
-    let client = SchemaRegistryClient::new();
-    on_record_written(do_nothing);
+  // Initialize the Schema Registry client
+  let client = SchemaRegistryClient::new();
 
-    // Use the client
+  // Register your transform function.
+  on_record_written(my_transform);
 }
 
-// Required callback, even if it does nothing
-fn do_nothing(_event: WriteEvent, _writer: &mut RecordWriter) -> anyhow::Result<()> {
+fn my_transform(_event: WriteEvent, _writer: &mut RecordWriter) -> anyhow::Result<()> {
+  // Use the client
```

Comment:
>// Use the client

Can't do a straight global like go. I'm sure there's a subjectively cleaner way to accomplish this, but here's what we do for integration testing:

https://github.com/redpanda-data/redpanda/blob/c7e27f720856b1787e4b26164863c3f7597acc1f/src/transform-sdk/rust/examples/schema_registry/main.rs#L31-L47